libraries/lib-wave-track/WaveClip.h

```diff
@@ -471,10 +512,18 @@ class WAVE_TRACK_API WaveClip final :
       bool committed{ false };
    };
 
+   // Real-time durations, i.e., stretching the clip modifies these.
```

Comment:
See uses of `@name` with `@{` and `@}` elsewhere in the source tree, so that Doxygen can associate a comment with multiple class members.